datumaro/components/extractor.py

```diff
@@ -106,10 +160,14 @@ def categories(self) -> CategoriesInfo:
     def get(self, id, subset=None) -> Optional[DatasetItem]:
         raise NotImplementedError()
 
+    def media_type(self):
+        return None
+
 class _ExtractorBase(IExtractor):
-    def __init__(self, length=None, subsets=None):
+    def __init__(self, length=None, subsets=None, media_type=Image):
```

Comment:
It is questionable if such default is reasonable. I can't say it is totally wrong, but it at least needs justification.

Reply:
I can see it a reasonable solution for the transition period, but then I'd like to see `None` here and even request this info from the extractor. Please think about the transition plan and document it somewhere. Maybe, create an issue to track this.